config/webpack/webpack.config.coverage.js

```diff
@@ -2,7 +2,8 @@
 /**
  * Webpack frontend test (w/ coverage) configuration.
  */
-var _ = require("lodash"); // devDependency
+var devRequire = require("builder-react-component-dev/require");
```

Comment:
would it make more sense to call this builderDevRequire or builderRequire? I know it's longer, but it might be easier to infer what's happening.